main/FatesHistoryInterfaceMod.F90

```diff
@@ -4221,2156 +4328,2613 @@ subroutine define_history_vars(this, initialize_variables)
 
 
     implicit none
-    
+
     class(fates_history_interface_type), intent(inout) :: this
     logical, intent(in) :: initialize_variables  ! are we 'count'ing or 'initializ'ing?
 
     integer :: ivar
-    character(len=10) :: tempstring 
-    
+    character(len=10) :: tempstring
+
     ivar=0
-    
+
```

Comment:
@adrifoster , could we add in the suffix key at the top here, so future authors have a quick reference on how to name output variables?

Reply:
yes, great idea. I will do that!

Reply:
looks good